src/luascript.cpp

```diff
@@ -2848,6 +2848,7 @@ void LuaScriptInterface::registerFunctions()
 	registerMethod(L, "Player", "isNearDepotBox", LuaScriptInterface::luaPlayerIsNearDepotBox);
 
 	registerMethod(L, "Player", "getIdleTime", LuaScriptInterface::luaPlayerGetIdleTime);
+	registerMethod(L, "Player", "setIdleTime", LuaScriptInterface::luaPlayerSetIdleTime);
```

Comment:
I think you can remove this functionality altogether using storages, doesn't seem to be used anymore within the engien